resources/language/en-US.ini

```diff
@@ -557,11 +593,10 @@ email_notifications.canvas_item_update_cta = "Click here to see it"
 
 email_notifications.canvas_board_item_created = "A new hypothesis was added to one of your canvases"
 email_notifications.canvas_item_created_message = "%1$s updated the item '%2$s'"
-email_notifications.canvas_item_update_cta = "Click here to see it"
```

Comment:
Why remove these? (They say the same thing, but I wanted to keep them in case we wanted to change the content)

Reply:
This was a duplicate as well. I got some duplicates after merging the main feature branch back in.